common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -581,4 +581,14 @@ trait PrebidSwitches {
     exposeClientSide = true
   )
 
+  val youtubePfpAdTargeting: Switch = Switch(
+    group = Commercial,
+    name = "commercial-youtube-pfp-ad-targeting",
+    description = "YouTube's PfP ad targeting parameters",
+    owners = group(Commercial),
+    safeState = Off,
+    sellByDate = new LocalDate(2019, 10, 1),
```

Comment:
Do we need  a `sellByDate` for this feature switch?

Reply:
I think once the PfP work is proven (following Monday's launch), this will become the default implementation and won't need the switch anymore. Alternatively we might find it is best left behind a switch, but this short sell-by date looks good to me to prompt that decision.